src/schemaLoader.js

```diff
@@ -40,24 +77,57 @@ function schemaLoader() {
     }
   });
 
-  return { typeDefs: allTypeDefs, resolvers: allResolvers };
+  return { typeDefs: mergeTypeDefs(allTypeDefs), resolvers: allResolvers };
 }
 
-const { typeDefs, resolvers } = schemaLoader();
-
 /**
- * Create executable schema from type definitions and resolvers
+ * Gets an executable schema that is transformed
+ * according to the permissions of the smaug client.
  */
-export const internalSchema = makeExecutableSchema({
-  typeDefs,
-  resolvers,
-});
+export async function getExecutableSchema({
+  loadExternal = true,
+  clientPermissions = permissions.default,
```

Comment:
skal vi have en eller enden form for clientPermission for os udviklere ?

Reply:
Ja, jeg tænkte at man kunne køre {admin: true}